src/api/api.ts

```diff
@@ -0,0 +1,34 @@
+import { Todo, User } from '../types/type';
+
+// eslint-disable-next-line
+const API_TODOS_URL = `https://mate.academy/students-api/todos`;
+// eslint-disable-next-line
+const API_USERS_URL = `https://mate.academy/students-api/users`;
+
+export async function getTodos(): Promise<Todo[]> {
+  try {
+    const response = await fetch(API_TODOS_URL);
+
+    if (response.ok) {
+      return await response.json();
+    }
+
+    throw new Error('ERROR Todos');
+  } catch (error) {
+    throw new Error(`There is a problem: ${error}`);
```

Comment:
There's no reason to throw an error, catch it in the same function and throw another on in the `catch` block.
Throwing first one is okay, but `catch` block here is useless.
Add it in the component.